### DISCUSSION TITLE
How to get Rmarkdown output from pointblank package

### DISCUSSION THREAD

**Discussion by hackR-gh** [rstudio/pointblank#288] [Q&A] [answered]
Unable to generate a html file when using the pointblank package and the scan_data() function.

In the help file for the pointblank package, it states: "_The output HTML report will appear in the RStudio Viewer and can also be integrated in R Markdown HTML output. If you need the output HTML as a string, it's possible to get that by using as.character() (e.g., scan_data(tbl = mtcars) %>% as.character()). The resulting HTML string is a complete HTML document where Bootstrap and jQuery are embedded within_"

So I tried this

```
write.csv(mtcars, "mtcars.csv")
fil <- "mtcars.csv"
map(fil, function(x) scan_data(read_csv(x), sections = "OVIMS"))
```

but only got a blank html file with info from the YAML header.

I also tried this:

`as.character(map(fil, function(x) scan_data(read_csv(x), sections = "OVIMS")))`

Any suggestions?

**Comment by rich-iannone** [comment, marked as answer]
There’s a new function called `export_report()` that should make this possible (https://rich-iannone.github.io/pointblank/reference/export_report.html). It’s currently in the development version of the package. Try it out and let me know if it doesn’t solve your problem.
